packages/patterns-react/src/patterns/LogoGrid/README.md

```diff
@@ -0,0 +1,100 @@
+# Logo Grid
+
+> The Logo Grid pattern is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+@import '@carbon/ibmdotcom-styles/scss/patterns/logo-grid/index.scss';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { LogoGrid } from '@carbon/ibmdotcom-patterns-react';
+import 'yourapplication.scss';
+
+const title =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.';
+
+const logosGroup = [
+  {
+    title: 'PHILIPS',
+    imgSrc: 'https://via.placeholder.com/140',
+    altText: 'placehoder',
+  },
+  {
+    title: 'charles SCHWAB',
```

Comment:
Slightly nitpicky, but not sure if we should mention actual companies in our open source code. You can probably use `IBM`, or `Company A` and `Company B`.